README.md

```diff
@@ -142,6 +142,15 @@ docker run --rm --network host --env-file .env -v ./.nodes:/root/.nodes \
    wavs-middleware -m mirror deploy
 ```
 
+List Mirror Operators:
+
+```bash
+# View stake registry status, including registered operators and their weights
+docker run --rm --network host --env-file .env -v ./.nodes:/root/.nodes \
```

Comment:
does this need the WAVS_SERVICE_MANAGER address? how does it know which operators to list on the mirror (i.e. where is the mirror?)

Reply:
Add SOURCE_SERVICE_MANAGER and MIRROR_SERVICE_MANAGER